### PR TITLE
gha: Fix gha actions

### LINKED ISSUE #7149: fix gha metrics yaml

### PR DESCRIPTION
**Pull request by GabyCT**
This PR removes an unrecognized value located in one of the yamls for the gha in order to make it work the CI again.

Fixes #7149

### CODE REVIEW

**Comment by GabyCT** [comment]
The current error in the gha without this PR is
```
[Invalid workflow file: .github/workflows/ci-on-push.yaml#L58](https://github.com/kata-containers/kata-containers/actions/runs/5327980899/workflow)
The workflow is not valid. In .github/workflows/ci-on-push.yaml (Line: 58, Col: 11): Error from called workflow kata-containers/kata-containers/.github/workflows/run-metrics.yaml@90e36f43ff8c60c4d7e27551212b6ad68fd5e66b (Line: 24, Col: 17): Unrecognized named-value: 'ARTIFACTS_DIR'. Located at
```

File: .github/workflows/run-metrics.yaml [hunk 1]
```diff
@@ -21,10 +20,6 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: kata-static-tarball-amd64${{ inputs.tarball-suffix }}
-          path: ${{ ARTIFACTS_DIR }}
```
Comment (dborquez) [inline .github/workflows/run-metrics.yaml:24]: could you hold `path:` w/ the literal value?
`path: kata-artifacts`

**Review by dborquez — APPROVED** [review]

File: .github/workflows/run-metrics.yaml (hunk 1 above)
Comment (dborquez) [inline .github/workflows/run-metrics.yaml:24]: could you please replace the variable name 'ARTIFACTS_DIR ' with the value on that variable? 
`path: kata-artifacts`

File: .github/workflows/run-metrics.yaml [hunk 2]
```diff
@@ -21,10 +20,6 @@ jobs:
...
-
-      - name: Install kata
-        run: bash tests/metrics/gha-run.sh install-kata ${{ ARTIFACTS_DIR }}
```
Comment (dborquez) [inline .github/workflows/run-metrics.yaml:27]: could you please replace the variable name 'ARTIFACTS_DIR ' with the value on that variable?
`run: bash tests/metrics/gha-run.sh install-kata kata-artifacts`

**Review by dborquez — CHANGES_REQUESTED** [review]

**Comment by GabyCT** [comment]
@dborquez changes applied

**Review by dborquez — APPROVED** [review]
lgtm, thank you @GabyCT

**Review by chavafg — APPROVED** [review]

**Pull request merged**